AlphaWallet/Transfer/Controllers/TransactionConfigurator.swift

```diff
@@ -52,7 +52,7 @@ class TransactionConfigurator {
 
         self.configuration = TransactionConfiguration(
             gasPrice: min(max(transaction.gasPrice ?? GasPriceConfiguration.default, GasPriceConfiguration.min), GasPriceConfiguration.max),
-            gasLimit: transaction.gasLimit ?? GasLimitConfiguration.default,
+            gasLimit: transaction.gasLimit ?? GasLimitConfiguration.max,
```

Comment:
Looks like `GasLimitConfiguration.default` isn't used anymore. Shall we remove it?